samples/emoji-search/ios/shared/src/commonMain/kotlin/app/cash/zipline/samples/emojisearch/exposed.kt

```diff
@@ -38,6 +39,7 @@ fun exposedTypes(
   treehouseUIKitView: TreehouseUIKitView<*>,
   uiViewRedwoodLayoutWidgetFactory: UIViewRedwoodLayoutWidgetFactory,
   uiViewRedwoodTreehouseLazyLayoutWidgetFactory: UIViewRedwoodTreehouseLazyLayoutWidgetFactory<*>,
+  redwoodUIScrollViewFactory: RedwoodUIScrollViewFactory,
```

Comment:
I think we can remove this and inline the `HostView` creation in `UIViewFlexContainer`.

Reply:
Updated